Makefile

```diff
@@ -48,4 +48,9 @@ sort:
 
 .PHONY: test
 test:
-	python manage.py test
+	pytest \
+		--cov=builder \
+		--cov=codelists \
+		--cov=coding_systems \
+		--cov=mappings \
+		--cov=opencodelists \
```

Comment:
1) Is there a good reason this isn't in the `pyproject.toml`?
2) Could we have `--source .` instead?  This pick up the right files, unless you've got a virtual environment in the same directory, in which case that could be omitted explicitly.